config-sets/config02/config-init.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#use sudo su command and hit enter to give root permissions
+#run the file with the command sh config-init.sh
+
+
+#xml starlet will be installed with the Y option selected through the shell script for the question Do you want to continue?
+
+if dpkg -l | grep "xmlstarlet"; then
```

Comment:
Shall we use some other way to verify if xmlstarlet installed? dpkg will not working on some environments.
we can use 
```sh
command -v xmlstarlet
```

Reply:
updated the script with command -v xmlstarlet